### DISCUSSION TITLE
Semantic Web Integration

### DISCUSSION THREAD

**Discussion by dougcallaway** [OvertureMaps/data#323] [Discussion]
What are your thoughts on integrating with the [Semantic Web](https://en.wikipedia.org/wiki/Semantic_Web)? There's significant similarity with well-established projects such as [Schema.org](https://schema.org/) and [Linked Open Data](https://lod-cloud.net/) projects like [DBpedia](https://www.dbpedia.org/). It seems there would be enormous mutual benefits in linking the projects with OvertureMaps data, including:
* Improving AI and search engine results by using existing Schema.org schemas like [Place](https://schema.org/Place) (see [Google Knowledge Graph](https://en.wikipedia.org/wiki/Google_Knowledge_Graph))  
* Enriching web map features with related data from the rest of the web, e.g. joining the administrative feature for Rome with DBpedia's [About:Rome](https://dbpedia.org/page/Rome); note that OSM already provides [links via RDF](https://osm2rdf.cs.uni-freiburg.de/)
* Conversely, linking from Semantic Web entities to OvertureMaps GERS IDs to provide richer spatial context (is anyone else underwhelmed by the static overview maps in Wikipedia?)
* Enabling advanced spatial reasoning queries with tools like [GeoSPARQL](https://en.wikipedia.org/wiki/OGC_GeoSPARQL)

In other words, why duplicate work/data that is already on the web? What other benefits can people think of?

**Comment by Spothedog1** [comment]
I've been doing this on my own by mapping columns to semantic web properties. Would love to have official collaboration.

Another very useful link would be to add [Data Commons IDs](https://www.datacommons.org/) to divisions so we could instantly link tons of public datasets.

@dougcallaway Interested in setting up a call and talking about it more?

**Reply by dougcallaway** [reply]
I'd be great to see how you're doing this already @Spothedog1. Any examples you're able to share? I'm not officially connected to OvertureMaps, but interested in getting involved, especially around linking public datasets as you mentioned. Happy to meetup on a call in the near future as well. You can reach me on [LinkedIn](https://www.linkedin.com/in/dougcallaway/) or doug.callaway@toponex.net.

**Comment by jetgeo** [comment]
@Spothedog1 and @dougcallaway: I would be happy to participate in discussions on this topic as well. I have been doing some research on OWL and RDF for geospatial information a couple of years ago, in particular, deriving ontologies from standardised UML models within ISO/TC 211 and OGC. A similar approach has been used by buildingSmart to go from EXPRESS to OWL for the BIM standard IFC, and could most likely also be used for going from YAML to OWL for Overture. The derived ontologies are not as good as those developed from scratch, as they have a flavour of UML or EXPRESS, but they are a good starting point. 
I have already done some work on Python scripts for deriving ISO 19109-compliant UML models from the Overture YAML schemas to enable comparison with standardised models such as INSPIRE in Europe. This is part of a research project we are working on for automated freight transport and the data requirements for those vehicles. I believe the scripts can be adapted to convert to OWL in addition to UML. 
We seem to be living in quite distant parts of the world, but I am used to working globally and at inconvenient times.

**Reply by dougcallaway** [reply]
@jetgeo, sounds like you have much more experience with this than me! I took a Semantic Web course in university ~8 years ago, but haven't seen much progress in the real world, except for Europe. Are you familiar with [GeoSPARQL](https://en.wikipedia.org/wiki/OGC_GeoSPARQL)? It looks really powerful, but interest seems to have dropped off the last couple years, so not clear if it's still relevant or useful in the context of OvertureMaps.

I'll see what I can do about getting us all on a call to discuss further.
